include/cells/cell.tcc

```diff
@@ -513,7 +597,10 @@ inline Eigen::Matrix<double, 3, 1> mpm::Cell<3>::local_coordinates_point(
       xi(1) = 2. * (point(1) - centre(1)) / ylength;
       xi(2) = 2. * (point(2) - centre(2)) / zlength;
 
-    } else {
+    }
+
```

Comment:
Can you remove the empty lines in between the if statements?

Reply:
I mean please remove empty lines in between the if, else if, and else statement, e.g.
```
if{
...
}
else if{
...
}
else{
...
}
```
not
```
if{
...
}
(empty line)
else if{
...
}
(empty line)
else{
...
}
```